react-native-in-app-payments-quickstart/app/service/ChargeCustomerCard.tsx

```diff
@@ -17,7 +17,7 @@
 import { CHARGE_CUSTOMER_CARD_SERVER_URL } from '../Constants';
 import ChargeCustomerCardError from '../ChargeCustomerCardError';
 
-export default async function chargeCustomerCard(customer_id, customer_card_id) {
+export default async function chargeCustomerCard(customer_id: any, customer_card_id: any) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it